test/sequential/sequential.status

```diff
@@ -46,3 +46,11 @@ test-watch-mode-inspect: PASS, FLAKY
 [$arch==s390x]
 # https://github.com/nodejs/node/issues/41286
 test-performance-eventloopdelay: PASS, FLAKY
+
+[$arch==ppc]
```

Comment:
This may need to be 
```suggestion
[$arch==ppc64]
```

Reply:
Do we have ppc64 though? It seems there's only ppc from https://github.com/nodejs/node/blob/1b74aa3cb2540e0be6f17f5b9dc57dd702929563/tools/utils.py#L83

Reply:
Actually I can't find where x64 is defined...maybe we can try ppc64 and see if it works

Reply:
ugh, no I think you're right and the arch gets changed to `ppc`. Which is kind of the opposite to https://github.com/nodejs/node/blob/1b74aa3cb2540e0be6f17f5b9dc57dd702929563/tools/getarch.py#L6-L10 but it's the `utils.py` version that is used in the Python test runner.

Reply:
> Actually I can't find where x64 is defined...maybe we can try ppc64 and see if it works

Now I'm more confused... this is setting arch based on `process.arch` from the `node` binary: https://github.com/nodejs/node/blob/1b74aa3cb2540e0be6f17f5b9dc57dd702929563/tools/test.py#L1676-L1688